src/input/ytdl_src.rs

```diff
@@ -31,8 +31,9 @@ const YOUTUBE_DL_COMMAND: &str = if cfg!(feature = "youtube-dlc") {
 /// If you need looping or track seeking, then consider using
 /// [`Restartable::ytdl`].
 ///
-/// Uses `youtube-dlc` if the `"youtube-dlc"` feature is enabled.
-///
+/// `youtube-dlc` and `yt-dlp` are also available by enabling the `youtube-dlc`
+/// and `yt-dlp` features respectively.
+/// 
```

Comment:
You've left a trailing space on this line: either remove it, or just run `cargo +nightly fmt`.

Reply:
I've fixed it xD